### ISSUE TITLE
C++ Bug: Whitespace Shown In Tables When 'Show White Spaces' Is Disabled

### ISSUE THREAD

**Issue by metal450** [giuspen/cherrytree#1071]
With "Preferences->Rich Text->Show White Spaces" disabled, whitespace is still shown in the cells of tables.

C++ version on Windows, current master built from source.

**Comment by txe** [comment]
`Rich Text->Show White Spaces` is for whitespace in rich text nodes.
`Plain Text & Code -> Show White Spaces` is for whitespace in plain text nodes, codeboxes and tables (honestly, I was confused too). Also, changes for codeboxes and tables are applied after cherrytree restart.

**Comment by metal450** [comment]
You're right, if I disable it for "plaintext & code," then it goes off for tables in Rich Text nodes.

(I actually deliberately keep whitespace on in code only, for the sake of noticing tab/spaces inconsistencies.  Tables on the other hand are used in writing content, which is why it was intended to be turned off - so that tables match the other Rich Text surrounding them).

**Comment by giuspen** [comment]
The thing is that currently the table is like a grid of codeboxes. We can improve the configurability in future
